packages/cli-kit/src/path.ts

```diff
@@ -22,7 +22,7 @@ export function relativize(path: string): string {
   const result = commondir([path, process.cwd()])
   const relativePath = relative(process.cwd(), path)
   const relativeComponents = relativePath.split('/').filter((component) => component === '..').length
-  if (result === '/' || relativeComponents > 2) {
+  if (result === '/' || relativePath === '' || relativeComponents > 2) {
```

Comment:
How does this behave in windows? the root path there would be `C:/` (or some other lette) i guess?